v2/tools/generator/internal/astmodel/allof_type_test.go

```diff
@@ -103,16 +103,6 @@ func TestAllOfAsTypePanics(t *testing.T) {
 	}).To(PanicWith(MatchError(expectedAllOfPanic)))
 }
 
-func TestAllOfAsDeclarationsPanics(t *testing.T) {
```

Comment:
minor: Either keep (if my above comment is correct and this should still panic), or transform into a test that ensures that it returns the right message when it errors?